x-pack/test/security_solution_cypress/cypress/tasks/hosts/uncommon_processes.ts

```diff
@@ -9,6 +9,9 @@ import { UNCOMMON_PROCESSES_TABLE } from '../../screens/hosts/uncommon_processes
 import { REFRESH_BUTTON } from '../../screens/security_header';
 
 export const waitForUncommonProcessesToBeLoaded = () => {
-  cy.get(UNCOMMON_PROCESSES_TABLE).should('exist');
+  // for some reason, the refresh button's aria-label is not updated here, so adding a check to make sure
+  // it's not disabled also should ensure that the page has correctly refreshed and prevent flaky tests
   cy.get(REFRESH_BUTTON).should('not.have.attr', 'aria-label', 'Needs updating');
```

Comment:
Nice fix! 😄 

Looking at this test I think this line could probably be removed as it's verifying on aria-label, which is not a good practice. It's not adding anything to the test and it could just introduce flakyness down the line.

What do you think?

Reply:
For this one we could yes, the problem is for [this guy](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/security_header.ts#L30) which is used in so many Cypress tests changing it scares me.

Reply:
Removed in https://github.com/elastic/kibana/pull/230523/commits/96f203c5703ab8a8bab1795790aac409c691fb9f